spark/src/main/scala/org/apache/spark/sql/comet/CometScanExec.scala

```diff
@@ -132,7 +132,7 @@ case class CometScanExec(
   lazy val bucketedScan: Boolean = wrapped.bucketedScan
 
   override lazy val (outputPartitioning, outputOrdering): (Partitioning, Seq[SortOrder]) =
-    (wrapped.outputPartitioning, wrapped.outputOrdering)
+    (UnknownPartitioning(wrapped.inputRDD.getNumPartitions), wrapped.outputOrdering)
```

Comment:
What the output partitioning is when it fails the test? I assume that the `outputPartitioning` should have correct partition number but not?

Reply:
Correct. `wrapped.outputPartitioning` was zero

Reply:
It is somehow weird. It means the `outputPartitioning` info is not correct in original ScanExec and not matched the output RDD's partition number. 🤔